Weather/src/sass/blocks/_widget.scss

```diff
@@ -0,0 +1,92 @@
+.widget {
+    padding: 95px 93px 95px 94px;
+    width: 800px;
+    height: 600px;
+    background: url(../img/main.jpg) center (center/cover) no-repeat;
+
+    &__content {
+        display: flex;
+        flex-wrap: wrap;
+        width: 615px;
+        height: 410px;
+    }
+
+    &-item {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        flex-wrap: wrap;
+        width: 205px;
+        height: 205px;
+
+        &__icon {
+            margin: 46px 58px 26px 57px;
+
+            &_margin {
+                margin-top: 40px;
+                margin-bottom: 25px;
+            }
+
+            &_cloudwindmoon {
+                margin: 24px 74px 17px 68px;
+            }
+        }
+
+        &__city {
+            margin-bottom: 37px;
+            color: #ffffff;
+            font-size: 16px;
+            font-weight: 700;
+            line-height: 11.52px;
+            text-transform: uppercase;
+        }
+
+        &__wrapper {
+            display: flex;
+            justify-content: space-evenly;
```

Comment:
Value `space-evenly` is not supported in ie11. You can check CSS properties and their values ​​for cross-browser compatibility here https://caniuse.com/. You can use `space-around` or `space-between` instead.